### PR TITLE
Initial Itto Implementation

### PR DESCRIPTION
**Pull request by Aftermathrar**
Lots of brute forcing to get the frames values needed.

### CODE REVIEW

**Comment by srliao** [comment]
is there any way to pull out the zhongli and geo traveler bug fixes from this and merge those first?

**Comment by HanSeoulOh** [comment]
Itto review checklist
- [ ] A2 (fine because no understanding of atkspd currently)
- [x ] A4
- [ ] C1
- [ ] C2
- [ ] C4
- [ ] C6

File: internal/characters/itto/abil.go [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+	// Check if used as Dasshu
+	if c.NormalCounter > 0 {
+		c.dasshuUsed = true
+		c.dasshuCount = c.NormalCounter
+	}
+
+	c.Core.Combat.QueueAttack(ai, core.NewDefCircHit(1, false, core.TargettableEnemy), f, f+travel, cb)
+	c.sCACount = 0
+
```
Comment (HanSeoulOh) [inline internal/characters/itto/abil.go:161]: I believe you're missing setting the CD in Itto's skill

**Review by HanSeoulOh — CHANGES_REQUESTED** [review]

**Review by HanSeoulOh — APPROVED** [review]

**Review by HanSeoulOh — APPROVED** [review]
lgtm

**Comment by HanSeoulOh** [comment]
worked https://gcsim.app/viewer/share/fSBTHCYz-_3gNZopzCE59

**Pull request merged**